core/services/keeper/registry1_1_synchronizer_test.go

```diff
@@ -55,6 +55,8 @@ func mockRegistry1_1(
 ) {
 	registryMock := cltest.NewContractMockReceiver(t, ethMock, keeper.Registry1_1ABI, contractAddress)
 
+	ethMock.On("HeaderByNumber", mock.Anything, mock.Anything).
+		Return(&types.Header{Number: big.NewInt(10)}, nil)
```

Comment:
curious, is this needed? do your changes affect registry1_1synchronizer?

Reply:
this is bc we moved the check for opts *bind.CallOpts in `GetActiveUpkeepIDs` but in 1.1 test, we did not mock this call.